Web/starter_project_barbies/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { BsFacebook, BsLinkedin, BsInstagram } from 'react-icons/bs'
+import Image from "next/image";
+
+type TeamCardProps = {
+  name: string
+  jobTitle: string
+  description: string
+  image: string
+  facebook: string
+  linkedin: string
+  instagram: string
+}
+
+const TeamCard = ({
+  name,
+  jobTitle,
+  description,
+  image,
+  facebook,
+  linkedin,
+  instagram,
+}: TeamCardProps) => {
+  return (
+    <div className="team-card box-content font-sans w-72 mt-4 text-center shadow-md shadow-gray-200 rounded-md py-2 m-auto">
+      <div className="team-card__image container overflow-hidden my-4">
+        <Image className="rounded-full mx-auto w-32" width={200} height={200} src={image} alt={name} />
+      </div>
+
+      <div className="team-card__content m-4">
+        <h3 className="team-card__name text-2xl m-3 font-extrabold tracking-widest uppercase">
+          {name}
+        </h3>
+        <p className="team-card__job-title m-3 tracking-widest text-base uppercase">
+          {jobTitle}
+        </p>
+        <p className="team-card__description text-sm mt-5 mb-3 text-gray-400 font-light px-4 tracking-wide">
+          {description}
+        </p>
+      </div>
+
+      <hr className="h-0.5 border-t-0 bg-neutral-100 opacity-100 dark:opacity-50 block m-auto w-5/6" />
+
+      <div className="team-card__social flex-row mt-3">
+        <div className="team-card__social__icon inline-block w-1/3">
```

Comment:
can you do these in a loop?
i think the social media fields list can be a part of the team member object defined to be a list of links (each link having an icon and a url)

implement it assuming not everyone has the same set of links (even if it is realistically not the case most of the time)

Reply:
@YiseBoge I have tried to make it as you suggested, but since we are using React-Icons I wasn't able to just pass icon string (like in the ones font-awesome provides.) If you have another way of going about it, let me know.

Reply:
ok you don't need to pass strings then, pass the element as an object

Reply:
alternatively you can also just keep your own key value pair of which icon is for which social media link (assuming we want the social media to come from database and is savable)